readthedocs/rtd_tests/utils.py

```diff
@@ -108,6 +108,21 @@ def make_test_hg():
     return directory
 
 
+def apply_fs(tmpdir, contents):
```

Comment:
I borrowed this from https://github.com/rtfd/readthedocs-build/blob/master/readthedocs_build/testing/utils.py p:

Reply:
ha, good point, but I feel a little unsure about that because we are going to re-integrate the `build` project :thinking:

Reply:
I think that's a problem for the future :) and we will need to remove this function from here anyway, hehe.